src/utils/array-utils.js

```diff
@@ -0,0 +1,14 @@
+export const removeCurrentPaintingFromRecommendedList = (
```

Comment:
I know how much you love for loops and this totally works, but since you mentioned you are out of touch with array methods you can also use filter here, and since filter creates a copy for you, you wouldnt need to dupliate the array. Kind of a 2 in 1 solution:
```
return allPaintingsResponse.filter(({name}) => name !== paintingName ).splice(0,3)
```